examples/carbon-relay-ng.ini

```diff
@@ -54,6 +54,9 @@ listen_addr = "0.0.0.0:2003"
 ### Pickle Carbon ###
 pickle_addr = "0.0.0.0:2013"
 
+### Server socket idle timeout in seconds ###
+server_socket_timeout = 3600
```

Comment:
Do we want to turn this on in the example config file?

Reply:
I think it is wise to enable by default.

Reply:
the position of this parameter is confusing. it's in between the pickle and amqp inputs, but it's actually a global option used by both pickle and plaintext carbon.
I think best would be to add the option to each input that uses it. e.g. to pickle and plain should have their own independent option

Reply:
I can split the config option, but since the plaintext and pickle inputs are both at the top level, both options should be at the top level as well (nesting these option might break older config file structure).

How about this 
Option1:
```
## Inputs ##                                                                                                                                                                                                        
                                                                                                                                                                                                                    
### plaintext Carbon ###                                                                                                                                                                                            
listen_addr = "0.0.0.0:2003"                                                                                                                                                                                        
plaintext_socket_timeout = 3600                                                                                                                                                                                     
                                                                                                                                                                                                                    
### Pickle Carbon ###                                                                                                                                                                                               
pickle_addr = "0.0.0.0:2013"                                                                                                                                                                                        
pickle_socket_timeout = 3600                                                                                                                                                                                        
                                                                                                                                                                                                                    
### AMQP ###     
[amqp]    
...
```

Would that work?

Another option could be
Option2:
```
## Inputs ##                                                                                                                                                                                                        
                                                                                                                                                                                                                    
### plaintext Carbon ###                                                                                                                                                                                            
[plaintext]                                                                                                                                                                                                         
listen_addr = "0.0.0.0:2003"                                                                                                                                                                                        
socket_timeout = 3600                                                                                                                                                                                               
                                                                                                                                                                                                                    
### Pickle Carbon ###                                                                                                                                                                                               
[pickle]                                                                                                                                                                                                            
pickle_addr = "0.0.0.0:2013"                                                                                                                                                                                        
socket_timeout = 3600                                                                                                                                                                                               
                                                                                                                                                                                                                    
### AMQP ###    
[amqp]
...    
```

But I'm concerned that it changed the config format and that existing config will break without conversion.

Which one do you prefer?